soc/arm/ti_simplelink/cc13x2_cc26x2/Kconfig.defconfig.series

```diff
@@ -10,12 +10,20 @@ source "soc/arm/ti_simplelink/cc13x2_cc26x2/Kconfig.defconfig.cc*"
 config SOC_SERIES
 	default "cc13x2_cc26x2"
 
+#config SYS_CLOCK_HW_CYCLES_PER_SEC
```

Comment:
Instead append a new conditional default. Same for the next changes below too.

Example,
default xxxx if BLE_CC13XX_CC26XX

Reply:
I actually don't think it will be possible to use the systick as the system timer. It's something I toyed with early on, but it's not very accurate in terms of waking up the core. There are a number of other peripherals that probably also need the RTC as well. It *is* multi-channel though, with up to 3 channels (that number is programmatically available as well). However, currently there seems to be some issues with Device Tree around shared irqs. Hopefully @vanti has some ideas about that too.